docs/fr/docs/advanced/path-operation-advanced-configuration.md

```diff
@@ -2,7 +2,7 @@
 
 ## ID d'opération OpenAPI
 
-!!! Attention
+!!! warning "Attention !"
```

Comment:
It would probably be better to have this fix in another PR, that would be easier to approve and merge even if it's very small.